_drafts/i2c-overview.md

```diff
@@ -0,0 +1,402 @@
+---
+title: "I2C in a Nutshell"
+description:
+  ""
+tag: [buses]
+author: francois
+image: 
+---
+
+<!-- excerpt start -->
+I2C is perhaps the most commonly used bus to connect ICs together. As such,
+firmwae engineers encounter it on most project. In this post, we explain how I2C
+works and go through common bugs and how to investiage them.
+<!-- excerpt end -->
+
+{:.no_toc}
+
+## Table of Contents
+
+<!-- prettier-ignore -->
+* auto-gen TOC:
+{:toc}
+
+
+## Why use I2C
+
+I2C has many advantages:
+1. It is cheap to implement, with fewer pins than SPI, and a simpler physical
```

Comment:
providing a little more detail what SPI or CAN are or that you are comparing similar transports might be helpful here for context

Reply:
Added wikipedia links, and a note about them being comparable low-speed buses. Don't want to add too much more (article is long enough as it is)